plugin-server/src/config/config.ts

```diff
@@ -131,6 +131,20 @@ export function getDefaultConfig(): PluginsServerConfig {
             ? 1024 // NOTE: ~1MB in dev or test, so that even with gzipped content we still flush pretty frequently
             : 1024 * 50, // ~50MB after compression in prod
         SESSION_RECORDING_REMOTE_FOLDER: 'session_recordings',
+        SESSION_RECORDING_REDIS_OFFSET_STORAGE_KEY: '@posthog/replay/partition-high-water-marks',
+        POSTHOG_SESSION_RECORDING_REDIS_HOST: undefined,
+        POSTHOG_SESSION_RECORDING_REDIS_PORT: 6379,
```

Comment:
nit: should be `undefined` here if your intent is to inherit the common redis config

Reply:
Yeah possibly that make sense